framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -232,7 +235,7 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 				await this._saveDataOnNewBlock(newBlockHeader);
 				await this._initializeReceivingChainClient();
 				this.logger.error(
-					error,
+					{ error: (error as Error).message },
```

Comment:
```suggestion
					{ err: (error as Error) },
```
will show the message well